### PR TITLE
[Wisp] WispThreadCompositeData.getCompositeData() should handle null WispTask

### PR DESCRIPTION
**Pull request by zhengxiaolinX**
Summary: WispThreadCompositeData.getCompositeData() uses LockSupport.getBlocker(), which would throw NPE to break its process.

Test Plan: com/alibaba/wisp/thread/TestThreadInfo.java

Reviewed-by: D-D-H, lvfei

Issue: #345

### CODE REVIEW

**Comment by zhengxiaolinX** [comment]
Please wait for the GHA checks before approval.

**Review by D-D-H — APPROVED** [review]
LGTM

**Review by Accelerator1996 — APPROVED** [review]

**Pull request merged**
